CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+This project is open to contributors, and we'd like to make it as easy as possible to share your ideas, and your code.
+
+Firstly, no contribution is too small. If you see something that's not right, please feel free to submit a spontaneous PR. It will very likely be reviewed same day.
+
+## Where to start
+
+If you don't know where to start, have a look at the list of [issues](https://github.com/tomarrell/lbadd/issues). If you find something interesting, feel free to say so in a comment, and you can be assigned to the issue.
```

Comment:
Would a relative link be better? In case this project grows and we decide to create an organization, we don't have to touch every file with a link in it.